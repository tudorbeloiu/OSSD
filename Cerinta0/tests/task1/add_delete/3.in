40
1
3
36
395
216
455
44
62
1
2
240
282
183
372
3
216
1
4
106
158
238
464
110
160
79
380
1
10
18
311
35
218
53
312
116
32
216
466
192
368
133
163
58
469
70
410
205
292
3
106
3
240
1
6
4
133
232
414
254
137
163
446
154
493
136
63
3
232
1
9
222
442
130
176
139
394
62
313
55
19
248
412
239
77
3
105
34
319
3
58
1
2
24
215
149
230
1
5
114
185
244
435
47
467
221
296
152
112
3
55
3
154
3
238
3
152
1
2
242
94
220
359
3
163
3
53
1
4
162
172
88
484
232
58
203
489
3
183
1
10
126
205
86
278
63
166
168
191
173
326
112
279
68
335
72
351
23
385
48
156
1
1
30
469
3
220
3
221
3
86
3
216
1
6
32
283
143
208
8
352
193
323
29
195
175
404
1
9
129
467
233
103
109
479
92
148
90
283
207
242
163
112
104
360
179
352
3
79
3
110
3
23
1
7
236
454
79
194
251
227
118
157
231
297
97
227
91
310
3
79
3
179
1
6
14
229
111
350
154
428
42
318
33
437
156
92
3
126
3
35
3
4
