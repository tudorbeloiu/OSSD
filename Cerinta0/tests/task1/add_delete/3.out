36: (0, 49)
216: (50, 106)
44: (107, 114)
240: (115, 150)
183: (151, 197)
36: (0, 49)
44: (107, 114)
240: (115, 150)
183: (151, 197)
106: (50, 69)
238: (198, 255)
110: (70, 89)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
116: (90, 93)
216: (410, 468)
192: (469, 514)
133: (515, 535)
58: (536, 594)
70: (595, 646)
205: (647, 683)
36: (0, 49)
110: (70, 89)
116: (90, 93)
44: (107, 114)
240: (115, 150)
183: (151, 197)
238: (198, 255)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
58: (536, 594)
70: (595, 646)
205: (647, 683)
36: (0, 49)
110: (70, 89)
116: (90, 93)
44: (107, 114)
183: (151, 197)
238: (198, 255)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
58: (536, 594)
70: (595, 646)
205: (647, 683)
4: (50, 66)
232: (684, 735)
254: (115, 132)
163: (736, 791)
154: (792, 853)
136: (94, 101)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
183: (151, 197)
238: (198, 255)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
58: (536, 594)
70: (595, 646)
205: (647, 683)
163: (736, 791)
154: (792, 853)
222: (854, 909)
130: (684, 705)
139: (910, 959)
62: (960, 999)
55: (67, 69)
248: (0, 0)
239: (133, 142)
3: (706, 719)
34: (0, 0)
36: (0, 49)
4: (50, 66)
55: (67, 69)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
238: (198, 255)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
163: (736, 791)
154: (792, 853)
222: (854, 909)
139: (910, 959)
62: (960, 999)
24: (536, 562)
149: (563, 591)
114: (1000, 1023)
244: (0, 0)
47: (0, 0)
221: (0, 0)
152: (720, 733)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
238: (198, 255)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
152: (720, 733)
163: (736, 791)
154: (792, 853)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
238: (198, 255)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
152: (720, 733)
163: (736, 791)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
152: (720, 733)
163: (736, 791)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
163: (736, 791)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
242: (198, 209)
220: (210, 254)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
242: (198, 209)
220: (210, 254)
79: (256, 303)
18: (304, 342)
35: (343, 370)
53: (371, 409)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
183: (151, 197)
242: (198, 209)
220: (210, 254)
79: (256, 303)
18: (304, 342)
35: (343, 370)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
162: (371, 392)
88: (720, 780)
232: (143, 150)
203: (781, 842)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
242: (198, 209)
220: (210, 254)
79: (256, 303)
18: (304, 342)
35: (343, 370)
162: (371, 392)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
126: (151, 176)
86: (0, 0)
63: (177, 197)
168: (0, 0)
173: (0, 0)
112: (0, 0)
68: (0, 0)
72: (0, 0)
23: (0, 0)
48: (0, 0)
30: (0, 0)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
79: (256, 303)
18: (304, 342)
35: (343, 370)
162: (371, 392)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
79: (256, 303)
18: (304, 342)
35: (343, 370)
162: (371, 392)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
79: (256, 303)
18: (304, 342)
35: (343, 370)
162: (371, 392)
216: (410, 468)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
79: (256, 303)
18: (304, 342)
35: (343, 370)
162: (371, 392)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
32: (210, 245)
143: (393, 418)
8: (419, 462)
193: (0, 0)
29: (0, 0)
175: (0, 0)
129: (0, 0)
233: (0, 0)
109: (0, 0)
92: (0, 0)
90: (0, 0)
207: (0, 0)
163: (0, 0)
104: (0, 0)
179: (0, 0)
36: (0, 49)
4: (50, 66)
110: (70, 89)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
32: (210, 245)
18: (304, 342)
35: (343, 370)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
32: (210, 245)
18: (304, 342)
35: (343, 370)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
32: (210, 245)
18: (304, 342)
35: (343, 370)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
236: (246, 302)
79: (0, 0)
251: (0, 0)
118: (67, 86)
231: (0, 0)
97: (0, 0)
91: (0, 0)
36: (0, 49)
4: (50, 66)
118: (67, 86)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
32: (210, 245)
236: (246, 302)
18: (304, 342)
35: (343, 370)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
118: (67, 86)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
126: (151, 176)
63: (177, 197)
242: (198, 209)
32: (210, 245)
236: (246, 302)
18: (304, 342)
35: (343, 370)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
14: (0, 0)
111: (0, 0)
154: (0, 0)
42: (0, 0)
33: (0, 0)
156: (0, 0)
36: (0, 49)
4: (50, 66)
118: (67, 86)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
63: (177, 197)
242: (198, 209)
32: (210, 245)
236: (246, 302)
18: (304, 342)
35: (343, 370)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
4: (50, 66)
118: (67, 86)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
63: (177, 197)
242: (198, 209)
32: (210, 245)
236: (246, 302)
18: (304, 342)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
36: (0, 49)
118: (67, 86)
116: (90, 93)
136: (94, 101)
44: (107, 114)
254: (115, 132)
239: (133, 142)
232: (143, 150)
63: (177, 197)
242: (198, 209)
32: (210, 245)
236: (246, 302)
18: (304, 342)
162: (371, 392)
143: (393, 418)
8: (419, 462)
192: (469, 514)
133: (515, 535)
24: (536, 562)
149: (563, 591)
70: (595, 646)
205: (647, 683)
130: (684, 705)
3: (706, 719)
88: (720, 780)
203: (781, 842)
222: (854, 909)
139: (910, 959)
62: (960, 999)
114: (1000, 1023)
